src/test/resources/projects/gradle/singleModGradleMP/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'war'
-    id 'io.openliberty.tools.gradle.Liberty' version '3.6.2'
+    id 'io.openliberty.tools.gradle.Liberty' version '3.8.3'
```

Comment:
The problem should also exist in `sampleGradleMPLSApp`.

Reply:
Changed `sampleGradleMPLSApp` also.